mono/mini/unaligned.cs

```diff
@@ -108,26 +133,51 @@ public static unsafe int test_0_ldind ()
 			ptr [i] = (byte)i;
 
 
-		if (Intrinsics.UnalignedLdInd2 (ptr + 0) != 0x0100)
-			return 1;
+		if (BitConverter.IsLittleEndian) {
 
-		if (Intrinsics.UnalignedLdInd2 (ptr + 1) != 0x0201)
-			return 2;
+			if (Intrinsics.UnalignedLdobj<short> (ptr + 0) != 0x0100)
```

Comment:
@nealef this changes here from `Intrinsics.UnalignedLdInd2` to `Intrinsics.UnalignedLdobj<short>`, I assume that's a copy-paste error?